test/tools/Modules/WebListener/WebListener.psm1

```diff
@@ -0,0 +1,147 @@
+Enum WebListenerProtocol 
+{
+    HTTP
+    HTTPS
+}
+
+Class WebListener 
+{
+    [int]$HttpPort
+    [int]$HttpsPort
+    [System.Management.Automation.Job]$Job
+
+    WebListener () { }
+
+    [string] ToString() 
+    {
+        return ('Port: {0}; Status: {1}' -f $This.Port, $This.JobStateInfo.State)
+    }
+
+    [String] GetStatus() 
+    {
+        return $This.JobStateInfo.State
+    }
+
+}
+
+[WebListener]$WebListener
+
+function Get-WebListener 
+{
+    [CmdletBinding(ConfirmImpact = 'Low')]
+    [OutputType([WebListener])]
+    param ()
+
+    process 
+    {
+        return [WebListener]$Script:WebListener
+    }
+}
+
+function Start-WebListener 
+{
+    [CmdletBinding(ConfirmImpact = 'Low')]
+    [OutputType([WebListener])]
+    param 
+    (
+        [ValidateRange(1,65535)]
+        [int]$HttpPort = 8083,
+
+        [ValidateRange(1,65535)]
+        [int]$HttpsPort = 8084
+    )
+    
+    process 
+    {
+        $runningListener = Get-WebListener
+        if ($null -ne $runningListener -and $runningListener.GetStatus() -eq 'Running')
+        {
+            return $runningListener
+        }
+
+        $initTimeoutSeconds  = 15
+        $appDll              = 'WebListener.dll'
+        $serverPfx           = 'ServerCert.pfx'
+        $serverPfxPassword   = 'password'
+        $initCompleteMessage = 'Now listening on'
+        
+        
+        $timeOut = (get-date).AddSeconds($initTimeoutSeconds)
+        $Job = Start-Job {
+            $path = Split-Path -parent (get-command WebListener).Path
+            Push-Location $path
+            dotnet $using:appDll $using:serverPfx $using:serverPfxPassword $using:HtttpPort $using:HtttpsPort
```

Comment:
`$using:HtttpsPort` -> `$using:HttpsPort` and `$using:HtttpPort` -> `$using:HtttpPort` should fix the build errors.